### PR TITLE
Refactored project structure

### PR DESCRIPTION
**Pull request by pancudaniel7**
- Changed project name
- Added project categories folders
- Moved files

### CODE REVIEW

File: Models/TestModel.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace Revitalise
+{
+    public class TestModel
+    {
+
+    }
+}
```
Comment (pancudaniel7) [inline Models/TestModel.cs:7]: This is my shitty code

File: Revitalise.csproj [hunk 2]
```diff
@@ -48,9 +48,9 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="AutoJoinWallsWithColumns.cs" />
-    <Compile Include="AutoJoinWallsWithWalls.cs" />
+    <Compile Include="Components/AutoJoinWallsWithColumns.cs" />
+    <Compile Include="Components/AutoJoinWallsWithWalls.cs" />
```
Comment (samuelpancu94) [inline Revitalise.csproj:52]: Q: 
- WTF yayo

Comment (pancudaniel7) [reply Revitalise.csproj:52]: Code example:
`WTF yoyoyo`

Comment (samuelpancu94) [reply Revitalise.csproj:52]: `FilteredElementCollector bucket = new FIlteredElementCollector();`

**Review by samuelpancu94 — APPROVED** [review]

**Pull request merged**